src/components/WebcamCapture/WebcamCapture.js

```diff
@@ -21,9 +18,13 @@ class WebcamCapture extends Component {
 			if (screenshot === null) {
 				reject('Camera not available.');
 			} else {
-				this.urlToImg(screenshot).then(img => {
-					this.urlToImg(this.cropImage(img)).then(resolve);
-				});
+				this.urlToImg(screenshot)
```

Comment:
Not entirely introduced in this PR but I think this could be simplified a bit:

```
const screenshot = this.webcam.current.getScreenshot();

if (screenshot === null) {
  return Promise.reject(new Error('Camera not available'));
}

return this.urlToImg(screenshot).then(img => this.urlToImg(this.cropImage(img))
```